planet/Decoration.ocd/Village.ocd/Bag.ocd/Script.c

```diff
@@ -2,6 +2,10 @@
 	@author Dustin Neß (dness.de)
 */
 
-protected func Construction() {
-SetProperty("MeshTransformation", Trans_Mul(Trans_Rotate(RandomX(-180,180),0,10), Trans_Scale(RandomX(130,200))));
-}
\ No newline at end of file
+public func Definition(proplist def) {
+	def.MeshTransformation = Trans_Scale(165); // average scale
```

Comment:
How is 165 the average of RandomX(787, 1212)? (You changed the random values in the Construction() call)

Reply:
165 is the average of the old random range 130 to 200.
.165 * .787 ≈ .13
.165 * 1.212 ≈ .2

Reply:
Ah, well, blame the github display then :D. The line is displayed as

SetProperty("MeshTransformation", Trans_Mul(Trans_Rotate(RandomX(-180,180),0,10), Trans_Scale(RandomX(787,1212))

and I need to scroll right in order to see the , GetID().MeshTransformation));